Modules/TPC/run/tpcQCRawDigits_direct.json

```diff
@@ -38,7 +38,8 @@
           "mergeableOutput": "true",
           "NRawDigitsNBins": "100",  "NRawDigitsXMin": "0", "NRawDigitsXMax": "100",
           "QmaxNBins":       "200",  "QmaxXMin":       "0", "QmaxXMax":       "200",
-          "TimeBinNBins":    "1000", "TimeBinXMin":    "0", "TimeBinXMax":    "100000"
+          "TimeBinNBins":    "1000", "TimeBinXMin":    "0", "TimeBinXMax":    "100000",
+          "OccupancyNBins":    "1000", "OccupancyXMin":    "0", "OccupancyXMax":    "100000"
```

Comment:
Shouldn't occupancy be between 0 and 1?

Reply:
Yes, I updated also the files to set as max 1